### PR TITLE
Migrate logout to twig

### LINKED ISSUE #944: newUI: Twigification of modules/core/templates/logout-iframe.php
This old template needs Twigification.. It's the last one to go and it's a leftover from #717!

### PR DESCRIPTION
**Pull request by jaimeperez**
This resolves #944.

@tvdijen, @thijskh, would you mind taking a look into this to see if I am missing something or something breaks?

### CODE REVIEW

File: modules/core/templates/logout-iframe.twig [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          </button>
+        </form>
+      </div>
+    </div>
+  {% endif %}
+{% endblock %}
+
+{% block postload %}
+
+    <script type="application/javascript" src="{{ asset('js/logout.js') }}"></script>
```
Comment (thijskh) [inline modules/core/templates/logout-iframe.twig]: `type=` no longer required in HTML5

File: templates/IFrameLogoutHandler.twig [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+{% extends "base.twig" %}
+{% block header %}{% endblock header %}
+{% block footer %}{% endblock footer %}
+{% block content %}
+
+          <i id="data"{% for k,v in data %} data-{{ k }}="{{ v }}"{% endfor %}></i>
+{% endblock content %}
+{% block postload %}
+
+    <script type="application/javascript" src="{{ asset('js/logout.js') }}"></script>
```
Comment (thijskh) [inline templates/IFrameLogoutHandler.twig]: As above

**Comment by tvdijen** [comment]
Yay, one step closer to 1.18 🤘

**Comment by jaimeperez** [comment]
Great! Thanks guys! I've fixed the script tags.

Tim, you can essentially set up at least three SPs to login to the same IdP, then trigger SLO from one of them. I've also tested missing SLO endpoint, wrong SLO endpoint, and now I'm working on a non-JS-friendly version, which should work as well.

**Pull request merged**